region/wreckedship/main/Bowling Alley.json

```diff
@@ -1506,6 +1529,52 @@
         "Screw Attack is still needed to clear the bomb blocks."
       ]
     },
+    {
+      "link": [3, 4],
+      "name": "Robot Kago, R-Mode Pause Abuse Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "f_DefeatedPhantoon",
+        {"or": [
+          "h_bombThings",
+          {"and": [
+            "ScrewAttack",
+            "Morph",
+            {"or": [
+              "Plasma",
+              {"and": [
+                {"notable": "Missile Chozo Camera Unlock"},
+                "canCameraManip",
+                "canHeroShot"
+              ]}
+            ]}
+          ]}
+        ]},
+        "h_CrystalFlashForReserveEnergy",
+        "h_shinechargeMaxRunway",
+        "canKago",
```

Comment:
I found kago to be the hardest way to do the strat.
1st - walljump right side and dboost into pause abuse - Extreme +
2nd - hijump right side and dboost into pause abuse - Extreme +
3rd - open spike pit and carefully time a jump out after reserves trigger - Insane